packages/sdk/src/react/_internal/transaction-machine/execute-transaction.ts

```diff
@@ -531,7 +547,7 @@ export class TransactionMachine {
 			} else if (step.id === StepType.tokenApproval) {
 				//TODO: Add some sort ofs callback heres
 				const hash = await this.executeTransaction(step);
```

Comment:
This will cause the `TransactionStatusModal` to be shown on token approvals, I dont think that is what we want.
Should we have a prop to disable it